src/pages/KyberDAO/KNCUtility/GasRefundBox.tsx

```diff
@@ -0,0 +1,342 @@
+                    >
+                      <Trans>Claim</Trans>
+                    </ButtonPrimary>
+                  )
+                ) : (
+                  <MouseoverTooltip
+                    text={
+                      <Trans>
+                        Gas Refund Rewards is only available on Ethereum chain. Switch your network to continue{' '}
+                        <LinkStyledButton onClick={openNetworkModal}>here</LinkStyledButton>
+                      </Trans>
+                    }
+                    width="244px"
+                  >
+                    <ButtonPrimary padding={upToXXSmall ? '8px 28px' : '8px 45px'} $disabled>
+                      <Trans>Claim</Trans>
+                    </ButtonPrimary>
+                  </MouseoverTooltip>
+                )
+              ) : (
+                <ButtonLight onClick={toggleWalletModal} padding="10px 12px">
+                  <Trans>Connect Wallet</Trans>
+                </ButtonLight>
+              )
+            ) : selectedTab === KNCUtilityTabs.Pending ? (
+              <Flex>
+                <Text fontSize={12} fontWeight={500} lineHeight="16px" as="span">
+                  <Trans>
+                    {' '}
+                    Available to claim in{' '}
+                    <Highlight>
+                      <StopWatch width={10} height={11} />
+                      {daoInfo
+                        ? readableTime(
+                            daoInfo.first_epoch_start_timestamp +
+                              daoInfo.current_epoch * daoInfo.epoch_period_in_seconds -
+                              Date.now() / 1000,
+                            2,
+                          )
+                        : '--:--:--'}
+                    </Highlight>
+                  </Trans>
+                </Text>
+              </Flex>
+            ) : null}
+          </Flex>
+        </RowBetween>
+      </Flex>
+      <Hr />
+      <RowBetween flexDirection="row" sx={{ gap: '16px' }}>
+        <Flex flexDirection="column" sx={{ gap: '16px' }}>
+          <TextDashed fontSize={14} lineHeight="20px" fontWeight={500} color={theme.subText}>
+            <MouseoverTooltip
+              width="fit-content"
+              text={<Trans>Total Gas Refund = Available + Pending + Claimed Gas Refund</Trans>}
+              placement="top"
+            >
+              <Trans>Total Gas Refund</Trans>
+            </MouseoverTooltip>
+          </TextDashed>
+          <Flex flexDirection="column" sx={{ gap: '8px' }}>
+            <Text fontSize={20} lineHeight="24px" fontWeight={500} color={theme.text} alignItems="center">
+              {account ? formattedNum(totalReward?.knc.toString() ?? '0') : '--'} KNC
+            </Text>
+            <Text fontSize={12} lineHeight="16px" fontWeight={500} color={theme.subText} alignItems="center">
+              {account
+                ? (totalReward?.usd ? '~' : '') + formattedNum(totalReward?.usd.toString() ?? '0', true)
```

Comment:
using formatDollarAmount for consistency

Reply:
`formattedNum` has used in 170 places, `formatDollarAmount` used in 111 places. So I dont think which one is better than which.

Reply:
that why we need to refactor this. haha

Reply:
Yes. Need a big refactor on this